priceCalculaterKata/priceCalculaterKata/Product.cs

```diff
@@ -0,0 +1,41 @@
+﻿class Product
+{
+    private String? name = null ;
+    private long upc;
+    private double price;
+    private double taxPercentage = 20  ;
+    
+
+    public String? Name {
+        get { return name ; }
```

Comment:
why you implemented the get and set methods with the default behavior ?
can't we just write
`public string name {get; set;}`